### PR TITLE
Ejm merchant invoice show 2

### PR DESCRIPTION
**Pull request by EMielke76**

### CODE REVIEW

File: app/controllers/merchant_invoices_controller.rb [hunk 1]
```diff
@@ -7,6 +7,6 @@ def index
   def show
     @merchant = Merchant.find(params[:merchant_id])
     @invoice = Invoice.find(params[:invoice_id])
-    @invoiced_item = InvoiceItem.where(invoice_id: params[:invoice_id])
+    #require "pry"; binding.pry
```
Comment (pleonar1) [inline app/controllers/merchant_invoices_controller.rb]: this can probably be done in a later pull request, but the pry might need to go eventually

**Review by pleonar1 — APPROVED** [review]
looks dope.  Other than some commented out stuff but that can be got on your next branch

File: app/controllers/merchant_invoices_controller.rb (hunk 1 above)
Comment (EMielke76) [reply app/controllers/merchant_invoices_controller.rb]: I'll kill it

**Review by jaw772 — APPROVED** [review]

**Review by LelandCurtis — APPROVED** [review]
looks good.

**Pull request merged**